cmd/kes/server.go

```diff
@@ -129,19 +130,25 @@ func server(args []string) error {
 		tlsCertPath = config.TLS.CertPath
 	}
 
-	switch {
-	case config.Keys.Fs.Path != "" && config.Keys.Vault.Endpoint != "":
-		return errors.New("Ambiguous configuration: FS and Hashicorp Vault endpoint specified at the same time")
-	case config.Keys.Fs.Path != "" && config.Keys.Aws.SecretsManager.Endpoint != "":
-		return errors.New("Ambiguous configuration: FS and AWS Secrets Manager endpoint are specified at the same time")
-	case config.Keys.Fs.Path != "" && config.Keys.Gemalto.KeySecure.Endpoint != "":
-		return errors.New("Ambiguous configuration: FS and Gemalto KeySecure endpoint are specified at the same time")
-	case config.Keys.Vault.Endpoint != "" && config.Keys.Aws.SecretsManager.Endpoint != "":
-		return errors.New("Ambiguous configuration: Hashicorp Vault and AWS SecretsManager endpoint are specified at the same time")
-	case config.Keys.Vault.Endpoint != "" && config.Keys.Gemalto.KeySecure.Endpoint != "":
-		return errors.New("Ambiguous configuration: Hashicorp Vault and Gemalto KeySecure endpoint are specified at the same time")
-	case config.Keys.Aws.SecretsManager.Endpoint != "" && config.Keys.Gemalto.KeySecure.Endpoint != "":
-		return errors.New("Ambiguous configuration: AWS SecretsManager and Gemalto KeySecure endpoint are specified at the same time")
+	keyStoreConfigCount := 0
+
+	if config.Keys.Fs.Path != "" {
+		keyStoreConfigCount++
+	}
+	if config.Keys.Vault.Endpoint != "" {
+		keyStoreConfigCount++
+	}
+	if config.Keys.Aws.SecretsManager.Endpoint != "" {
+		keyStoreConfigCount++
+	}
+	if config.Keys.Gemalto.KeySecure.Endpoint != "" {
+		keyStoreConfigCount++
+	}
+	if config.Keys.GCP.ProjectID != "" {
+		keyStoreConfigCount++
+	}
+	if keyStoreConfigCount > 1 {
+		return errors.New("Ambiguous configuration: multiple key store are specified at the same time")
```

Comment:
Looks like you have removed the context from the error, now we have to guess what the issue is about what was wrong with the previous per target errors?

Reply:
Yes, you're right. Let me revert this part.